apps/hash-agents/app/agents/template/__main__.py

```diff
@@ -38,4 +38,4 @@ def main(agent_input: Input) -> Output:
     setup()
 
     output = main(Input(expression="round(pi * 13.37)"))
-    logger.info(f"output: {output.result}")
+    logger.info(output=output)
```

Comment:
Should this be 
```suggestion
    logger.info(result=output.result)
```

(genuinely asking as I don't know what the shape of `output` is and haven't checked, but we were previously printing `output.result`)

Reply:
I don't mind, just noticed this changed the behaviour that's all